src/contexts/ClusterNodesMapContext/ClusterNodesMapContext.tsx

```diff
@@ -0,0 +1,32 @@
+import {type ReactNode, createContext, useEffect, useContext} from 'react';
+
+import type {NodesMap} from '../../types/store/nodesList';
+import {getNodesList, selectNodesMap} from '../../store/reducers/nodesList';
+import {useTypedDispatch, useTypedSelector} from '../../utils/hooks';
+
+type ClusterNodesMapContextProps = {
+    children: ReactNode;
+};
+
+const ClusterNodesMapContext = createContext<NodesMap | undefined>(undefined);
+
+/** NodesMap - almost constant cluster nodes to hosts mapping */
+export function ClusterNodesMapContextProvider({children}: ClusterNodesMapContextProps) {
```

Comment:
Why do we need context? Trigger `getNodesList` in the component that uses this provider and  loaded data will be in global store.